pkg/logs/input/container/docker.go

```diff
@@ -196,18 +194,18 @@ func (dt *DockerTailer) forwardMessages() {
 			// the decoder has been stopped, there is no more message to forward
 			return
 		}
-		ts, sev, updatedMsg, err := parser.ParseMessage(output.Content)
+		_, sev, updatedMsg, err := parser.ParseMessage(output.Content)
 		if err != nil {
 			log.Warn(err)
 			continue
 		}
 		containerMsg := message.NewContainerMessage(updatedMsg)
 		msgOrigin := message.NewOrigin()
 		msgOrigin.LogSource = dt.source
-		msgOrigin.Timestamp = ts
+		msgOrigin.Timestamp = time.Now().UTC().Format(config.DateFormat)
```

Comment:
Could we add a comment here? This means that we're not storing properly the last processed docker log's timestamp in the auditor. For simplification, we're fine with that, but it wouldn't hurt to add a small comment to keep that in mind in case we have a support case around this (for instance if people bounce the agent when we're delayed)

Reply:
👍